server/controllers/comment.controller.js

```diff
@@ -28,6 +28,7 @@ class CommentController {
     const { id } = req.user;
     const { slug } = req.params;
     const { comment, highlightedText, spanId } = req.body;
+    const commentType = req.body.commentType || null;
```

Comment:
hi @TheDrizzyWay is the `|| null` really necessary? Any particular reason for the null check?

Reply:
Its simply to avoid breaking the app, since the commentType is optional and has no middleware covering it.

Reply:
Alright